files/default/neutron-ha-tool.py

```diff
@@ -561,13 +563,14 @@ def replicate_dhcp(qclient, noop=False):
 
 
 def migrate_l3_routers_from_agent(qclient, agent, targets,
-                                  noop, wait_for_router, delete_namespace):
+                                  noop, wait_for_router, delete_namespace,
+                                  agent_picker):
```

Comment:
> My logic here was to add the new argument as the next arg

That's not what you did in the other methods :-) TBH, I don't care much as long as there's some consistency.